src/i18n-config.js

```diff
@@ -0,0 +1,65 @@
+import i18n from 'i18next';
```

Comment:
How come this file is in `src/`?

Reply:
Since it's not a component I moved it to src